text/0000-lts-releases.md

```diff
@@ -0,0 +1,109 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle releases with long term support (LTS)
+- Start Date: 2023-02-24
+- Author(s): joe-kimmel-vmw
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Lifecycle to define approximately 4 releases per year as LTS.
+LTS releases will have critical CVEs backported for a window of at least 1 year.
+LTS releases may also recieve other backports, including but not limited to golang patch version bumps.
+
+# Definitions
+[definitions]: #definitions
+
+**CVE**: While some scholars claim this acronym has origins in the Mayan codices[1], others seem to support a Norse origin[2]. Regardless of its hagiography, CVEs come in varying severities such as Critical, High, and "you probably don't need to patch it but we still wanted to let you know."
+These severity levels are issued by the shadowy council of security elders and cannot be modified by mere practitioners such as ourselves.
+However there are multiple councils[3] and they will occasionally grade the same vulnerability differently.
+This proposal only relates to CVEs graded Critical by at least one council.
+
+
+# Motivation
+[motivation]: #motivation
+
```

Comment:
I would like this section to be more fleshed out and list scenarios where we would recommend patches of older lifecycle versions rather than the latest.